author/topics/document-format/section-terms.adoc

```diff
@@ -637,6 +628,14 @@ The following keys are supported:
 
 `script`:: the script of the designation. Specified as an ISO 15924 code.
 
+`fieldOfApplication`:: text indicating the field of application of the term (typically the domain),
+and used to disambiguate the designation from other designations identical in the current language.
+The fieldOfApplication in ISO/IEC appears in angle brackets, after the designation
+itself [added in https://github.com/metanorma/metanorma-standoc/releases/tag/v1.11.2].
+
+`usageInfo`:: text used to disambiguate the designation from other designations identical in the current language,
+but not relating to the field of application of the term [added in https://github.com/metanorma/metanorma-standoc/releases/tag/v1.11.2].
+
```

Comment:
@opoudjis we need a consistent way of naming attributes. In the implementation you're using `fieldofapplication` (just lowercase), in documentation it is `fieldOfApplication` (lower camelcase). The AsciiDoc convention is hyphenation (`field-of-application`), and in XML we can also stick with `field-of-application`.

Before making this publicly available we ought to fix this.